content/id/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -0,0 +1,244 @@
+        role: myrole
+    spec:
+      containers:
+        - name: web
+          image: nginx
+          ports:
+            - name: web
+              containerPort: 80
+              protocol: TCP
+    ```
+
+2. Atur kubelet pada suatu Node untuk menggunakan manifes pada web ini dengan menjalankan menggunakan argumen `--manifest-url=<url-manifes>`. Pada Fedora, ubah pada `/etc/kubernetes/kubelet` untuk menambahkan baris ini:
+
+    ```
+    KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --manifest-url=<url-manifes>"
+    ```
+
+3. Jalankan ulang kubelet. Pada Fedora, kamu dapat menjalankan:
+
+    ```shell
+    # Jalankan perintah ini pada Node tempat kubelet berjalan
+    systemctl restart kubelet
+    ```
+
+## Mengobservasi perilaku Pod statis
+
+Ketika kubelet berjalan, secara otomatis akan menjalankan semua Pod statis yang terdefinisi.
+Ketika kamu mendefinisikan Pod statis dan menjalankan ulang kubelet, Pod statis yang baru
+akan dijalankan.
+
+Kamu dapat melihat Container yang berjalan (termasuk Pod statis) dengan menjalankan (pada Node):
+```shell
+# Jalankan perintah ini pada Node tempat kubelet berjalan
+docker ps
+```
+
+Hasilnya akan seperti berikut:
+
+```
+CONTAINER ID IMAGE         COMMAND  CREATED        STATUS         PORTS     NAMES
+f6d05272b57e nginx:latest  "nginx"  8 minutes ago  Up 8 minutes             k8s_web.6f802af4_static-web-fk-node1_default_67e24ed9466ba55986d120c867395f3c_378e5f3c
+```
+
+Kamu dapat melihat mirror Pod pada API server:
+
+```shell
+kubectl get pods
+```
+```
+NAME                       READY     STATUS    RESTARTS   AGE
+static-web-my-node1        1/1       Running   0          2m
+```
+
+{{< note >}}
+Pastikan kubelet memiliki izin untuk membuat Pod mirror pada server API. Jika tidak,
+pembuatannya akan ditolak oleh API server. Lihat
+[PodSecurityPolicy](/id/docs/concepts/policy/pod-security-policy/).
+{{< /note >}}
+
+
+{{< glossary_tooltip term_id="label" text="Label" >}} dari Pod statis
+akan dibuat juga pada mirror Pod. Kamu dapat menggunakan label tersebut
+seperti biasa menggunakan {{< glossary_tooltip term_id="selector" text="selectors" >}},
+atau yang lainnya.
+
+Kamu dapat mencoba untuk menggunakan kubelet untuk menghapus mirror Pod pada API server,
+namun kubelet tidak akan menghapus Pod statis:
+
+```shell
+kubectl delete pod static-web-my-node1
+```
+```
+pod "static-web-my-node1" deleted
+```
+Kamu akan melihat Pod tetap berjalan:
+```shell
+kubectl get pods
+```
+```
+NAME                       READY     STATUS    RESTARTS   AGE
+static-web-my-node1        1/1       Running   0          12s
+```
+
+Kembali ke Node tempat kubelet berjalan, kamu dapat mencoba menghentikan Container
+Docker secara manual.
+Kamu akan melihat, setelah beberapa saat, kubelet akan mengetahui dan akan menjalankan ulang Pod
+secara otomatis:
+
+```shell
+# Jalankan perintah ini pada Node tempat kubelet berjalan
+docker stop f6d05272b57e # ganti dengan ID pada Container-mu
```

Comment:
```suggestion
docker stop f6d05272b57e # ganti dengan ID pada kontainermu
```

same as above

Reply:
I think for simplicity we agreed on using Container for everything? @phanama

Reply:
Ah! Apology, didn't remember that 😅 
Resolving this for now.

Reply:
Just ignore the kontainers @wahyuoi